models/openocean/optimism/openocean_optimism_trades.sql

```diff
@@ -0,0 +1,98 @@
+{{ config(
+    tags = ['dunesql'],
+    schema = 'openocean_optimism',
+    alias = alias('trades'),
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address']
+    )
```

Comment:
By the way we may add a post_hook here to expose this model.
And `trace_address` may not be needed anymore.
```suggestion
    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index'],
    post_hook='{{ expose_spells(\'["optimism"]\',
                                "project",
                                "openocean",
                                \'["ARDev097"]\') }}'
    )
```